controllers/statusreport/statusreport_adapter.go

```diff
@@ -298,3 +311,16 @@ func (a *Adapter) getComponentSourceFromSnapshotComponent(snapshot *applicationa
 	}
 	return nil, fmt.Errorf("couldn't find the requested component source info in the given Snapshot")
 }
+
+// determineIfAllIntegrationTestsNotTriggered returns name of integrationTestScenario that is not triggered yet.
+func (a *Adapter) determineIfAllIntegrationTestsNotTriggered(integrationTestScenarios *[]v1beta1.IntegrationTestScenario, testStatuses *intgteststat.SnapshotIntegrationTestStatuses) string {
```

Comment:
nit: we might consider a different name for this function for more clarity, something like `findMissingIntegrationTestFromStatus` or similar. Not a hard requirement from me though.

Reply:
how about `findUntriggeredIntegrationTestFromStatus` ?

Reply:
Sure, that looks good to me too.

Reply:
Thanks!